src/views/pages/Dashboard.vue

```diff
@@ -64,7 +64,7 @@ const lineOption = ref({
     axisPointer: {
       type: 'cross',
       label: {
-        backgroundColor: '#6a7985'
+        backgroundColor: 'blue'
```

Comment:
Score: C

Reason: Hardcoded magic string 'blue' instead of a configurable or defined constant.

Suggestion: Define a constant for the background color or make it configurable through a theme or configuration object.